src/main/java/com/dmgrt/cinema/Main.java

```diff
@@ -74,24 +76,24 @@ public static void main(String[] args) throws AuthenticationException {
         inceptionSession.setShowTime(LocalDateTime.now().plusMonths(5));
         movieSessionService.add(inceptionSession);
 
-        System.out.println(movieSessionService.findAvailableSessions(2L, LocalDate.now()));
-        System.out.println(movieSessionService.findAvailableSessions(3L, LocalDate.now()));
+        logger.info(movieSessionService.findAvailableSessions(2L, LocalDate.now()));
+        logger.info(movieSessionService.findAvailableSessions(3L, LocalDate.now()));
 
         User jake = new User();
         jake.setPassword("tort");
         jake.setEmail("jake1956@meta.ua");
 
         jake = authenticationService.register(jake.getEmail(), jake.getPassword());
-        System.out.println("Registered: " + jake);
+        logger.info("Registered: " + jake);
 
         jake = authenticationService.login("jake1956@meta.ua", "tort");
```

Comment:
let's wrap this in try-catch and add log.warn with the informative message in `catch` block